tests/test_docs.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+import wasmer
+
+known_issues = {

```

Comment:
Anything that took me longer than 5 secs to understand, I skipped. When running pytest with `-rxXs` should [show the details](https://docs.pytest.org/en/latest/how-to/skipping.html).

Reply:
Good to me. I can try to fix them. The PR is already an awesome improvement!